docs/guide/persistence_cache.md

```diff
@@ -105,7 +120,11 @@ Redis is used via [Redis pecl extension](https://pecl.php.net/package/redis).
 See [Redis Cache Adapter in Symfony documentation](https://symfony.com/doc/3.4/components/cache/adapters/redis_adapter.html#configure-the-connection)
 for information on how to configure Redis.
 
-Example:
+Out of the box in `app/config/cache_pool/cache.redis.yml` you'll find a default example that can be used.
+
+For eZ Platform Cloud/Platform.sh: This is automatically configured in `app/config/env/platformsh.php` if you have enabled Redis as `rediscache` Platform.sh service.
+
+For anything else, you can enable it with environment variables detected automatically by `app/config/env/generic.php`. For instance, if you set the following environment variables `export CACHE_POOL="cache.redis" CACHE_DSN="secret@example.com:1234/13"`, it will result in config like this:
```

Comment:
```suggestion
For anything else, you can enable it with environment variables detected automatically by `app/config/env/generic.php`.
For instance, if you set the following environment variables `export CACHE_POOL="cache.redis" CACHE_DSN="secret@example.com:1234/13"`, it will result in config like this:
```